Jenkinsfile

```diff
@@ -1,2 +1,29 @@
-// Build the plugin using https://github.com/jenkins-infra/pipeline-library
-buildPlugin(jenkinsVersions: [null, '2.107.1'])
+#!/usr/bin/env groovy
+
+@Library('sec_ci_libs@v2-latest') _
+
+def master_branches = ["master", "usi-jenkins", ] as String[]
+
+ansiColor('xterm') {
+  // using shakedown node because it's a lightweight alpine docker image instead of full VM
+  node('shakedown') {
+    stage("Verify author") {
+      user_is_authorized(master_branches, '8b793652-f26a-422f-a9ba-0d1e47eb9d89', '#marathon-dev')
```

Comment:
I think #marathon-dev does not exist anymore. Do we have the same thing in our jenkins files?

Reply:
Good catch. I'll go with orchestration